spring-integration-amqp/src/main/java/org/springframework/integration/amqp/dsl/AmqpBaseOutboundEndpointSpec.java

```diff
@@ -311,4 +311,17 @@ public S errorMessageStrategy(ErrorMessageStrategy errorMessageStrategy) {
 		return _this();
 	}
 
+	/**
+	 * Set a timeout after which a nack will be synthesized if no publisher confirm has
+	 * been received within that time. Missing confirms will be checked every 50% of this
+	 * value so the synthesized nack will be sent between 1x and 1.5x this timeout.
+	 * @param timeout the approximate timeout.
+	 * @return the spec.
+	 * @since 5.3
+	 */
+	public S confirmTimeout(long timeout) {
```

Comment:
Right, this is one of those I'd like to cover with DSL extensions and all those `protected` ctors 😄